crates/vm/levm/src/gas_cost.rs

```diff
@@ -449,18 +453,40 @@ pub fn sstore(
     } else {
         let static_gas = SSTORE_STATIC;
 
+        let default_dynamic = match fork {
+            // https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1283.md
```

Comment:
What do you think of this?
```suggestion
            // https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1283.md
            // https://eips.ethereum.org/EIPS/eip-1283
```